third_party/pyth/price-service/README.md

```diff
@@ -29,15 +35,20 @@ $ docker run --platform linux/amd64 -d --network=host ghcr.io/certusone/guardian
 --network <guardianNetworkPathForMainNet> \
 ```
 
-Then to run the pyth_price_service docker container using a config file called
-${HOME}/pyth_price_service/env and logging to directory ${HOME}/pyth_price_service/logs, do the
+Then to run the pyth_price_service docker container using a config file called `${HOME}/pyth_price_service/env` and logging to directory `${HOME}/pyth_price_service/logs`, do the
 following:
 
 ```
 $ docker run \
 --volume=${HOME}/pyth_price_service:/var/pyth_price_service \
 -e PYTH_PRICE_SERVICE_CONFIG=/var/pyth_price_service/env \
 --network=host \
--d \
 pyth_price_service
 ```
+
+Or, run `docker compose up` in the `third_party/pyth/price-service` directory:
```

Comment:
```suggestion
Or, run docker compose in the `third_party/pyth/price-service` directory using the commands below. It will run a spy listening to the wormhole testnet network and a price service that listens to Pyth prices published to it.

```